.travis.yml

```diff
@@ -12,6 +12,13 @@ jobs:
       before_script: ./scripts/decrypt-app-credentials.sh
       script: ./scripts/test-hosting.sh
       after_script: skip
+  include:
+    - stage: emulator triggers end to end test
+      if: repo == head_repo OR type == push
+      node_js: "8"
+      before_script: skip
+      script: ./scripts/triggers-end-to-end-tests/test.js
```

Comment:
Right now fails with: 
```
$ ./scripts/triggers-end-to-end-tests/test.js
unable to read firebase.json Error: ENOENT: no such file or directory, open 'firebase.json'
```

I think we want to add a `./end-to-end.sh` and then have that look something like:
```
cd ./scripts/triggers-end-to-end-tests
node test.js
```

That will also allow us to add more tests over time